tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -762,6 +764,7 @@ datasets:
       clip_duration: 79
       temporal_stride: 1
       num_samples: 400
+      image_input: True
```

Comment:
possibly it is not so clear, but maybe better create parameter for getting 2 streams instead of changing converter logic for working with images?
Current solution is not backward compatible and may affect users.